### ISSUE TITLE
No module named 'setuptools.wheel' after installing

### ISSUE THREAD

**Issue by yomoko** [cuemacro/findatapy#17]
I can't install findatapy. It has the following error. Why is that? Thanks.
![image](https://user-images.githubusercontent.com/33768560/33792725-6e51ec0e-dc75-11e7-88ef-37e8ef1367f4.png)

**Comment by saeedamen** [comment]
Have you tried installing setuptools package? Eg. pip install setuptools

**Comment by yomoko** [comment]
@saeedamen  No, I think it came with anaconda. My setuptools version is 36.5.0.post20170921

**Comment by saeedamen** [comment]
@yomoko how have you tried to install findatapy (was it using pip install git+https://github.com/cuemacro/findatapy.git)?

**Comment by yomoko** [comment]
@saeedamen Yes, I did exactly what the instruction says. 
![image](https://user-images.githubusercontent.com/33768560/33897476-93035b6c-df33-11e7-8ac8-496617648c6d.png)
![image](https://user-images.githubusercontent.com/33768560/33897508-a763c39e-df33-11e7-88aa-edb714686eb7.png)
![image](https://user-images.githubusercontent.com/33768560/33897565-ca45972a-df33-11e7-9d2c-d4662074d865.png)

**Comment by yomoko** [comment]
I solved the problem by install Visual C++

**Comment by yomoko** [comment]
@saeedamen Is it possible to define dukascopy frequency as 4 hour, 2 hour so on? Now only min, daily is available. Also, is it possible to make the candle interval start at 17:00 New York time, instead of 00:00 GMT?

**Comment by whuhit** [comment]
pip install --upgrade setuptools

**Comment by saeedamen** [comment]
@yomoko sorry for the slow reply, I'm not sure if the underlying DukasCopy API allows downloading at different frequencies. You can however downsample the tick data using pandas, using .resample (and there's a variation of this for making OHLC bars too).

**Comment by INKWWW** [comment]
> pip install --upgrade setuptools



> pip install --upgrade setuptools

Please insist on executing this command until the terminal tells you 'Successfully installed setuptools-41.0.0 (the newest version at present)'   [ because I upgraded setuptools successfully by several times' attempts ]

**Comment by JerrryNie** [comment]
> pip install --upgrade setuptools

Thanks! It helped me a lot!
